src/test/components/MarkerTable.test.js

```diff
@@ -53,25 +81,18 @@ describe('MarkerTable', function() {
 
     expect(markers.toJSON()).toMatchSnapshot();
   });
-});
 
-/**
- * Mock out any created refs for the call tree components with relevant information.
- */
-function createNodeMock(element) {
-  const classNameParts = element.props.className.split(' ');
-  if (
-    // <VirtualList />
-    classNameParts.includes('treeViewBody') ||
-    // <VirtualListInner />
-    classNameParts.includes('treeViewBodyInner')
-  ) {
-    return {
-      addEventListener: () => {},
-      // Set an arbitrary size that will not kick in any virtualization behavior.
-      getBoundingClientRect: () => getBoundingBox(2000, 1000),
-      focus: () => {},
-    };
-  }
-  return null;
-}
+  it('renders some basic network markers', () => {
+    // These were all taken from real-world values.
+    const profile = getProfileWithMarkers([...NETWORK_MARKERS]);
+
+    const markers = renderer.create(
+      <Provider store={storeWithProfile(profile)}>
+        <Markers />
+      </Provider>,
+      { createNodeMock }
+    );
+
+    expect(markers.toJSON()).toMatchSnapshot();
```

Comment:
My assumption is that this test is supposed to assert that the marker table shows network markers. I don't think that a snapshot test is a good choice for creating this assertion. Snapshots so far have proved really helpful to assert a component has some kind of arbitrary shape for its HTML, and renders in a consistent way. The snapshots therefore are really good at showing that *something* has changed, and letting the author/reviewer see the diff of the changes. This is great for making sure that the rendered DOM elements are doing what you expect, without actually loading up the final product.

Snapshots that assert specific types of behavior are really hard to understand when they fail. For instance in this case, if the network markers no longer show up, the diff will consist of a lot of removed markup. This is a lot less clear than an assertion failing like:

```js
expect(wrapper.find(SOME_ROW_SELECTOR).length).toBe(NETWORK_MARKERS.length);
```

In addition, when we change the shape of the component's markup, this test will fail, and need to be updated with the new markup changes. This is noise to the author/reviewer, while the `MarkerTable renders some basic markers` test would be the only failing test, that helpfully shows that some markup changed. This should be a signal to the reviewer, as it's only asserting that the component has some kind shape.

I believe a more maintainable test here would be to convert the existing one to using `enzyme` rather than `react-test-renderer`, then rewrite the test. We wrote a lot of tests with `react-test-renderer` initially, and only added `enzyme` later, so many older tests still have not be converted.

I think we'll want to use full [Full DOM rendering](https://github.com/airbnb/enzyme/blob/master/docs/api/mount.md) here rather than shallow rendering, since we need the table to fully render. We have a bunch of existing tests that use enzyme that can be referenced for the conversion, although I'd be happy to do a pass converting the existing test to enzyme to make writing the new test easier if you want me to do that.